pytest_mpl/plugin.py

```diff
@@ -520,6 +518,14 @@ def compare_image_to_baseline(self, item, fig, result_dir, summary=None):
                 return error_message
 
         results = compare_images(str(baseline_image), str(test_image), tol=tolerance, in_decorator=True)
+
+        if ext in ['png', 'svg']:  # Use original files
+            summary['result_image'] = test_image.relative_to(self.results_dir).as_posix()
+            summary['baseline_image'] = baseline_image.relative_to(self.results_dir).as_posix()
```

Comment:
Can you move these lines back to were they were above and just have a `if ext not in ['png', 'svg']` for the two lines below? There are a couple of failure `return` points these lines have moved past. In SunPy, the baseline images are not committed when a new test is added (they are stored in a separate repo). So the `result_image` needs to be set so we can see an image for new tests in PRs.

Reply:
This should fix the tests

Reply:
Haha I just did that and then read your comment :)